src/components/Search/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Reminder, this should change after a rebase from `feature/product-list-ui` 😎 (Close this comment when it changes)